src/controllers/logincontroller.js

```diff
@@ -21,8 +21,13 @@ const logincontroller = function () {
     const user = await userprofile.findOne({ email: { $regex: "^" + escapeRegex(_email) + "$", $options: 'i' } })
       .catch(error => res.status(400).send(error));
 
-    // returning 403 if the user not found or the found user is inactive.
-    if (!user || user.isActive === false) {
+    // returning 403 if the user not found
+    if (!user) {
+      res.status(403).send({ message: 'Username not found.' });
+      return;
+    }
+    // returning 403 if the found user is inactive (Message is taken care of in PR438).
+    if (user.isActive === false) {
```

Comment:
```suggestion
    if (!user.isActive) {
```

Reply:
Thanks for the suggestion @joyceiphone ! I believe PR#438 is taking care of isActive status and a relevant message so I did not touch this line of code.

Reply:
@cvtqx Okay. Thank you for letting me know.